### DISCUSSION TITLE
Goerli Reth 0.1.0-alpha6 Stuck on IndexStorageHistory 8600000

### DISCUSSION THREAD

**Discussion by mckamyk** [paradigmxyz/reth#4134] [Help]
Hello, I've been attempting to sync a reth node on goerli next to a Prysm client and have been stuck on this stage for at least a week. Its been syncing for 18 days in total.

The only relevant log I've been able to find:
```
consensus::engine: Pipeline sync progress is inconsistent first_stage_checkpoint=9386062 inconsistent_stage_id=IndexStorageHistory inconsistent_stage_checkpoint=8600000
```
Whenever reth posts its current sync status, this log is shown
```
INFO reth::cli: Status connected_peers=16 stage=IndexStorageHistory checkpoint=8600000 eta=unknown
```
Ignore the low peer count, this was right after restarting, normally around 70.

I started metric scraping into grafana yesterday, along with some performance stats. Its got a steady-ish read of 5MB/s over the past 24 hours, but a flatline of 0 B/s writes, outside of 2 events that it wrote just 600 bytes.

What might be the reason its stuck?

**Comment by Charlotte-br560** [comment]
Your reth node on Goerli is encountering an issue with syncing, stuck explicitly on the stage of IndexStorageHistory. This could be due to various reasons, including network issues, disk I/O bottlenecks, or software bugs.

Given the logs you provided, the pipeline sync progress appears inconsistent, indicating a potential problem reaching checkpoints. The low peer count after restarting might also indicate connectivity issues.

The flatline of 0 B/s writes suggests that your node might be unable to write data correctly, which could further contribute to the syncing problem.

To troubleshoot this issue, I recommend checking your network connection, ensuring that your disk I/O is not overloaded, and looking into any potential bugs in the reth software. Additionally, monitoring system performance metrics such as CPU usage, disk utilization, and network traffic can provide valuable insights into the root cause of the problem.

If feasible, you may also consider using tools like Crawlbase to further analyze and troubleshoot your syncing issue.

**Reply by rakita** [reply]
Goerly is deprecated few months ago, will close this discussion
